### PR TITLE
Added a method in app.js for empty string

### PR DESCRIPTION
**Pull request by cosmoimai**
I have added a method like if empty string provided it will give details of that place

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "chalk": "^4.1.0",
     "ora": "^5.1.0",
     "postman-request": "^2.88.1-postman.25",
+    "request": "^2.88.2",
```
Comment (jmtellez) [inline package.json:27]: use postman-request

File: app.js [hunk 2]
```diff
@@ -6,15 +6,34 @@ const options = require("./utils/options");
 const geocode = require("./utils/geocode");
 const forecast = require("./utils/forecast");
 const menu = require("./utils/menu");
+const request = require("request");
+
+////////////////////////
+///////////////////////
```
Comment (jmtellez) [inline app.js:12]: unused module. If you need to make a request, use `postman-request` as request module is deprecated. postman-request is a fork of the same module

File: app.js [hunk 3]
```diff
@@ -6,15 +6,34 @@ const options = require("./utils/options");
...
+      forecast(latitude, longitude, (error, forecastdata) => {
+        if (error) {
+          return console.log(error);
+        }
+        console.log(forecastdata);
+      });
+    });
+
+    ////////////////
+    //spinner.fail("Provide a location");
```
Comment (jmtellez) [inline app.js:36]: `address` is not defined.

**Review by jmtellez — CHANGES_REQUESTED** [review]

File: app.js (hunk 3 above)
Comment (cosmoimai) [reply app.js:36]: ok, sir, I will try to resolve this

File: package.json (hunk 1 above)
Comment (cosmoimai) [reply package.json:27]: ok sir

File: app.js (hunk 2 above)
Comment (cosmoimai) [reply app.js:12]: ok sir

File: app.js (hunk 3 above)
Comment (cosmoimai) [reply app.js:36]: sir, I will do it by today

**Comment by cosmoimai** [comment]
Sir, What is process.env.MAPBOX doing because it is giving undefined

**Comment by jmtellez** [comment]
process.env.MAPBOX is getting the env var that stores the api key to Mapbox. You need this key and weatherstack key for the tool to work

**Pull request closed without merging**